src/app/adminLayout/admin-header/admin-header.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import {MenuItem} from "primeng/api";
+
+@Component({
+  selector: 'app-admin-header',
+  templateUrl: './admin-header.component.html',
+  styleUrls: ['./admin-header.component.css']
```

Comment:
według powyższego to też można by wyrzucić?

Reply:
no w tej chwili niby faktycznie css jest pusty, chociaż tak samo uważam, że jak kiedyś weźmiemy kredki i będziemy to kolorować to będziemy potrzebować te pliki css więc proponuję zostawić

Reply:
może zostać narazie do developmentu